standalone/jdaviz-cli-entrypoint.py

```diff
@@ -7,9 +7,24 @@
 import matplotlib_inline
 import matplotlib_inline.backend_inline
 
+# We still see the above error on CI on jdaviz, and the PyInstaller
+# output recommends the following:
+import matplotlib
+matplotlib.use("module://matplotlib_inline.backend_inline")
+# since matplotlib 3.9 (see https://github.com/matplotlib/matplotlib/pull/27948),
+# it seems that matplotlib_inline.backend_inline is an alias for inline
+# so we make sure to communicate that to PyInstaller
+matplotlib.use("inline")
+
 import jdaviz.cli
 
 
 if __name__ == "__main__":
     # should change this to _main, but now it doesn't need arguments
-    jdaviz.cli.main(layout="")
+    args = sys.argv.copy()
+    # change the browser to qt if not specified
+    if "--browser" not in args:
+        args.append("--browser")
+        args.append("qt")
```

Comment:
since qt is an optional dependency (and we want to keep it that way), can this default to qt only if it is installed and otherwise use the system browser?

Reply:
This is only for the standalone, which is built in CI with qt installed. Or do you mean if people build in locally?

Reply:
if anyone installs jdaviz (from pip for example) but not using the standalone installer, they still have access to the CLI, but may not have the qt optional dependencies.

Reply:
But they would not use this file right, maybe you are confusing it with cli.py?